arm/Microsoft.ServiceBus/namespaces/virtualNetworkRules/readme.md

```diff
@@ -0,0 +1,30 @@
+# ServiceBus Namespace Virtual Network Rules `[Microsoft.ServiceBus/namespaces/virtualNetworkRules]`
+
+This module deploys a virtual network rule for a service bus namespace.
+
+## Resource Types
+
+| Resource Type | Api Version |
+| :-- | :-- |
+| `Microsoft.ServiceBus/namespaces/virtualnetworkrules` | 2018-01-01-preview |
+
+## Parameters
+
+| Parameter Name | Type | Default Value | Possible Values | Description |
+| :-- | :-- | :-- | :-- | :-- |
+| `cuaId` | string |  |  | Optional. Customer Usage Attribution id (GUID). This GUID must be previously registered |
+| `name` | string | `[format('{0}-vnr', parameters('namespaceName'))]` |  | Optional. The name of the virtual network rule |
```

Comment:
:D Interesting default value. Should we add an issue for improving the default value output in the script that generates the readme?

Reply:
Yes, though I'm not sure right now how we could solve that. Check for [parameters]?